opba-protocols/xs2a-protocol-tests/xs2a-bdd-tests-common/src/main/java/de/adorsys/opba/protocol/xs2a/tests/e2e/stages/CommonGivenStages.java

```diff
@@ -58,9 +57,9 @@ public SELF rest_assured_points_to_opba_server(String opbaServerUri) {
     public SELF user_registered_in_opba_with_credentials(String username, String password) {
         RestAssured
                 .given()
-                    .header(X_REQUEST_ID, UUID.randomUUID().toString())
+                    .header(X_REQUEST_ID, DEFAULT_X_REQUEST_ID)
```

Comment:
Consent API calls should use generated request IDs (like it was before)

Reply:
random UUID is generated